src/test/java/org/broadinstitute/hellbender/utils/python/StreamingPythonExecutorIntegrationTest.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.hellbender.utils.python;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.SkipException;
+import org.testng.annotations.Test;
+
+import java.util.Collections;
+
+public class StreamingPythonExecutorIntegrationTest extends GATKBaseTest {
+
+    // This test validates that the StreamingPythonScriptExecutor will throw if the Python environment
+    // is not activated, and will not pass if the environment is activated. It has to be an integration
+    // test because unit tests are run on the Docker image, which has the Python environment activated.
+
+    @Test()
+    public void testRequirePythonEnvironment() {
+        // This test is deliberately left out of the "python" test group in order to ensure that
+        // it only executes when the Python environment has *NOT* been properly established. Also,
+        // skip this test if we're running on the Docker because the Python environment is always
+        // activated there.
+        final String isDockerCI = System.getenv("CI");
+        if (isDockerCI != null && isDockerCI.equalsIgnoreCase("true")) {
```

Comment:
ehh... this could be extracted out into a `skipIfNotInDockerImage()` since it's copied in two places.